pkg/agent/datamodel/types.go

```diff
@@ -143,17 +143,6 @@ type Distro string
 // Distro string consts.
 const (
 	Ubuntu                              Distro = "ubuntu"
-	Ubuntu1804                          Distro = "ubuntu-18.04"
-	Ubuntu1804Gen2                      Distro = "ubuntu-18.04-gen2"
-	AKSUbuntu1804Gen2                   Distro = "ubuntu-18.04-gen2" // same distro as Ubuntu1804Gen2, renamed for clarity
```

Comment:
there's also a file in RP which basically defines the same stuff, we should clean that up as well